internal/repotesting/repotesting.go

```diff
@@ -100,6 +100,9 @@ func (e *Environment) Close(ctx context.Context, t *testing.T) {
 		}
 	}
 
+	// may need to remove the maintenance lock
+	os.Remove(e.configFile() + ".mlock")
```

Comment:
For tests that run maintenance tasks, this file lingers around. Mmm, perhaps it should be removed on disconnect. Currently here to prevent the check below from failing a test.